.github/workflows/covector.yml

```diff
@@ -92,16 +92,6 @@ jobs:
         # tests on Windows 2019, until we can figure out a way to fix the problem.
         # NOTE: Using Ubuntu 18.04 to provide glibc compatibility. (#588)
         os: [ubuntu-18.04, ubuntu-latest, macos-latest, windows-2019]
-        node-version: ['14.x', '16.x', '18.x']
```

Comment:
I'm on mobile so I can't check the default version that `actions/setup-node` uses, but I think the step on line 106 needs to be updated if we remove this. On `wallet.rs` we use 16.x (although tbh, any LTS version of Node should work as long as the scripts don't break)

Reply:
As far as I see, wallet.rs uses 14.x, but maybe it's worth updating that one to 16.x as well. I set it here to 16.x.